doc/chapter-configuration.asciidoc

```diff
@@ -30,6 +30,22 @@ unbind-key       R
 bind-key         ^R    reload-all
 ----
 
+=== Splitting macros into multiple lines
+
+Macros defined in config files which are long can be split up into multiple
+lines using backslashes; these must be the last character on the line and will
+immediately concatenate it with the following line. It's important that _nothing_
+follows the \ on the same line, otherwise the \ character is treated "as is".
```

Comment:
Did you check that backslashes here are rendered correctly in HTML and the manpage? See how we typeset the `#` sign a few paragraphs above.

Reply:
Yeah the backslash turns out fine, but the other stuff did not (I used --- instead of ----). Changed in patch

Reply:
I still had my doubts, so I compiled it and the backslash didn't have the proper background in HTML, and wasn't underlined in the manpage ;) But it's easy enough to fix, lemme do it by writing a suggestion and committing immediately.

```suggestion
follows the `{backslash}` on the same line, otherwise the `{backslash}` character is treated "as is".
```